### PR TITLE
Quick Fix Alias Type Names

### PR DESCRIPTION
**Pull request by michaelweilsalesforce**
**Related issues**
When aliasing Type, the alias is sometimes lost in the reader.

**Describe the proposed solution**
Quick fix

### CODE REVIEW

File: readers/src/main/scala/com/salesforce/op/readers/Reader.scala [hunk 1]
```diff
@@ -66,7 +66,8 @@ private[readers] trait ReaderType[T] extends Serializable {
    * @param opParams contains map of reader type to ReaderParams instances
    * @return ReaderParams instance if it exists
    */
-  final def getReaderParams(opParams: OpParams): Option[ReaderParams] = opParams.readerParams.get(this.typeName)
+  final def getReaderParams(opParams: OpParams): Option[ReaderParams] = opParams.readerParams
+    .get(this.wtt.tpe.toString.split("\\.").last)
```
Comment (tovbinm) [inline readers/src/main/scala/com/salesforce/op/readers/Reader.scala]: This seems invalid, since it will only use the type name without package prefix.

Comment (michaelweilsalesforce) [reply readers/src/main/scala/com/salesforce/op/readers/Reader.scala]: Yes that's what we want. Example of reader param is `Map{"entity1" -> ...}`

Comment (tovbinm) [reply readers/src/main/scala/com/salesforce/op/readers/Reader.scala]: well it does match the check in the `readerInputTypes` then ;) I propose to use the full aliased name then `tpe.toString`

Comment (michaelweilsalesforce) [reply readers/src/main/scala/com/salesforce/op/readers/Reader.scala]: That would involve changing all our OpParams...

**Pull request merged**